### ISSUE TITLE
ARIA in HTML updates re: popovertarget and command attributes

### ISSUE THREAD

**Issue by scottaohara** [dequelabs/axe-core#4837]
[ARIA in HTML](https://github.com/w3c/html-aria/pull/514) is updating to identify misuse of `aria-expanded` when used on a button element that also have a `popovertarget` or `command` attribute specified.

These updates mirror other similar rules such as input type=checkbox discouraging use of `aria-checked` since the browser already handles these states and the use of author specified ARIA attributes is redundant at best, and can lead to potential misalignments that browsers _should_ correct for - though there have been issues with this in the past.

This [reduced test case](https://w3c.github.io/html-aria/tests/popover-and-command.html) helps illustrate the updates to the spec.

**Comment by straker** [comment]
@chutchins25 Fixing this should be updating [aria-allowed-attr-evaluate's ignoredAttrs fucntion](https://github.com/dequelabs/axe-core/blob/develop/lib/checks/aria/aria-allowed-attr-evaluate.js#L65) to look at if the `vNode` is a button element and has the `aria-expanded` attribute set to `'true'` and does not have the `popovertarget` or `command` attributes then return `true`. Adding a test case similar to the one Scott provided.
